app/src/main/java/com/keylesspalace/tusky/StatusListActivity.kt

```diff
@@ -152,6 +156,8 @@ class StatusListActivity : BottomSheetActivity(), HasAndroidInjector {
                     {
                         followTagItem?.isVisible = true
                         unfollowTagItem?.isVisible = false
+
+                        Snackbar.make(binding.root, getString(R.string.unfollowing_hashtag_success_format, tag), Snackbar.LENGTH_SHORT).show()
```

Comment:
That one got the "LONG" (only) for the action button.

Reply:
I meant the one on line 138, that's LENGTH_LONG with no action associated with the snackbar.

The success snackbars should persist for consistent lengths. Over in https://github.com/tuskyapp/Tusky/blob/develop/app/src/main/java/com/keylesspalace/tusky/components/notifications/NotificationsFragment.kt#L305-L322 I used `LENGTH_SHORT`, 

The error snackbars should persist indefinitely until the user dismisses them. This is so the user has the chance to note down the error (or take a screenshot) in case they need to report the problem (either to us, or to their server admin).

Reply:
Ok, changed that one to SHORT.

The one with the action remains LONG.